docs/workflow/blocks/load_schema.rst

```diff
@@ -89,8 +89,8 @@ If the above is loaded in (using a database / DB block, like the "editItem" flow
     }
 
 
-Reference transclusion
-----------------------
+Referencing / transcluding other schemas (via array / object config)
```

Comment:
I suspect some people will be put off by the "transclude" word. It's not one commonly used. We could benefit from a more general description.

"Referencing structures and data from other schemas"

Reply:
Yes, I've made a few of the titles and explanations more clear with less jargon right away, thank you.